src/jobManager.ts

```diff
@@ -34,6 +34,7 @@ export async function doJob() {
 export async function getJob() {
   _maybeExit();
   logger.info('Fetching job');
+  await registerTurtleVersion(turtleVersion);
```

Comment:
I think we should do it in the `main` function in `src/server.ts`. It's not possible to change the Turtle version without restarting the server.